docs/swagger/spec.yml

```diff
@@ -2233,6 +2199,38 @@ definitions:
           enabled:
             type: boolean
 
+  SingleBandOptions:
+    type: object
+    properties:
+      band:
+        type: number
+        format: integer
+      dataType:
+        type: string
+      blendMode:
+        type: string
+      colorScheme:
+        type: object
+      legendOrientation:
+        type: string
+
+  ColorCorrection:
+    type: object
+    properties:
+      isSingleBand:
+        type: boolean
+      singleBandOptions:
+        $ref: '#/definitions/SingleBandOptions'
+      rbgBandOptions:
```

Comment:
This seems a bit awkward to me because it's going to force a single-band correction to carry some baggage in the `rgbBandOptions` object that it will never need (and vice versa). I'm not sure what version of Swagger we're using but it might be possible to use `oneOf` here, which would allow us to replace these three fields with a single one. Alternatively, we might want to move the single-band corrections to a different endpoint and use a different data type rather than mixing the two together.